tests/integration/targets/web_app_pool/tasks/main.yml

```diff
@@ -0,0 +1,43 @@
+---
+# Cannot use win_feature to install IIS on Server 2008.
+# Run a brief check and skip hosts that don't support
+# that operation
+- name: check if win_feature will work on test host
+  ansible.windows.win_command: powershell.exe "Get-WindowsFeature"
+  register: module_available
+  failed_when: False
+
+# Run actual tests
+- block:
+  - name: ensure IIS features are installed
+    ansible.windows.win_feature:
+      name: Web-Server
+      state: present
+      include_management_tools: True
+    register: feature_install
+
+  - name: reboot after feature install
+    ansible.windows.win_reboot:
+    when: feature_install.reboot_required
+
+  - name: set version of IIS for tests
+    community.windows.win_file_version:
+      path: C:\Windows\System32\inetsrv\w3wp.exe
+    register: iis_version
+
```

Comment:
As mentioned in https://github.com/ansible-collections/microsoft.iis/pull/6#issuecomment-2516144683 we should try and avoid adding extra dependencies to our tests, especially `community.windows`. Luckily in this case we don't need this anymore as it was used to see if IIS 8 was present and since our base Windows version is 2016 we can guarantee that.

```suggestion
```